app/ui/src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts

```diff
@@ -256,16 +260,21 @@ export class DataMapperHostComponent extends FlowPage implements OnInit {
         atlasmapping: mappings ? mappings : ''
       },
       onSave: () => {
-        setTimeout(() => {
-          this.initializeMapper();
-        }, 10);
+        this.initializeMapper();
       }
     });
+    this.countDownInitialization();
   }
 
   initializeMapper() {
-    this.initialized = true;
-    this.initializationService.initialize();
+    if ( this.initializedCountDown == 0 ) {
```

Comment:
Please run `yarn lint:fix` to get your code properly linted and validated. We do not support extra spaces inside parens anymore. The lint:fix command will take care not only of auditing the code but to automatically fixing these issues.

Reply:
not really part of the original issue.